src/controllers/pools/deposits/list.controller.ts

```diff
@@ -2,12 +2,12 @@ import { Request, Response } from 'express';
 import DepositService from '@/services/DepositService';
 
 const controller = async (req: Request, res: Response) => {
+    console.log('pooladdress', req.params.address)
```

Comment:
@valeriagrazzini I guess it makes more sense (REST spec) to use query string parameters for this. I think for the members list something similar is implemented.